lib/loader.js

```diff
@@ -10,12 +10,15 @@ var glob = require('glob');
  */
 exports.get = function(file) {
   // If the file is a type natively recognized by node, just require it
-  if (_.contains(['.js', '.coffee', '.json'], path.extname(file))) {
+  if (_.includes(['.js', '.coffee', '.json'], path.extname(file))) {
     var content = {};
     try {
       content = require(file);
     } catch (e) {
-      return {};
+      console.log('The following error was thrown while trying to read', file);
```

Comment:
I've no idea how much refactoring it would take, but grunt has grunt.log.error that would make more sense to use than console.log.